src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala

```diff
@@ -16,7 +16,7 @@ trait Tables {
   /** DDL for all tables. Call .create to execute. */
```

Comment:
Why is this schema changing at all?

Reply:
Oh, I reran GenSchema because my local wasn't up to date.

Reply:
Also, this is just an untagged comment, so I'm sure it won't affect anything

Reply:
Looks like you dropped a primary key! That could hit performance a lot..

Reply:
Are you sure you ran the SQL from `conseil.sql`. We should not have any change in schema here.